packages/acceptance-tests/pkg-tests-specs/sources/pnp-esm.test.ts

```diff
@@ -141,57 +141,6 @@ describe(`Plug'n'Play - ESM`, () => {
     ),
   );
 
-  test(
-    `it should support named exports in commonjs files`,
-    makeTemporaryEnv(
-      {
-        dependencies: {
-          'no-deps-exports': `1.0.0`,
-        },
-        type: `module`,
-      },
-      async ({path, run, source}) => {
-        await expect(run(`install`)).resolves.toMatchObject({code: 0});
-
-        await xfs.writeFilePromise(
-          ppath.join(path, `index.js` as Filename),
-          `import {foo} from 'no-deps-exports';\nconsole.log(foo)`,
-        );
-
-        await expect(run(`node`, `./index.js`)).resolves.toMatchObject({
-          code: 0,
-          stdout: `42\n`,
-        });
-      },
-    ),
-  );
-
-  test(
-    `it should always set default as module.exports when importing a commonjs file`,
```

Comment:
Quick note: it's worth adding a (in-PR) comment when tests are removed 😃

Reply:
I wrote it on the commit that removed them